crypto/ml_dsa/ml_dsa_sign.c

```diff
@@ -44,11 +44,17 @@ static void signature_init(ML_DSA_SIG *sig,
 
 /*
  * FIPS 204, Algorithm 7, ML-DSA.Sign_internal()
+ * If msg_is_mu is 1 then msg contains a pre-computed M' otherwise
+ * M' is computed by concatenating enc_ctx and msg.
```

Comment:
M` in the standard refers to the encoded message i.e. 0X || ctx_len || ctx || msg. 
The value passed in when msg_is_mu is 1 is actually a precomputed mu.
i.e. H(Bytestobits(tr)||M`)

Reply:
Yeah, I have been a little sloppy there, will fix tomorrow.